docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,159 @@
+
+.. _sql-globally-available:
+
+Globally available statements
+-----------------------------
+
+The following statements are implemented in the core engine and available with
+any connector:
+
+* :doc:`/sql/call`
+* :doc:`/sql/deallocate-prepare`
+* :doc:`/sql/describe-input`
+* :doc:`/sql/describe-output`
+* :doc:`/sql/execute`
+* :doc:`/sql/explain`
+* :doc:`/sql/explain-analyze`
+* :doc:`/sql/prepare`
+* :doc:`/sql/reset-session`
+* :doc:`/sql/set-session`
+* :doc:`/sql/set-time-zone`
+* :doc:`/sql/show-functions`
+* :doc:`/sql/show-session`
+* :doc:`/sql/use`
+* :doc:`/sql/values`
+
+.. _sql-read-operations:
+
+Read operations
+---------------
+
+The following statements provide read access to data and meta data exposed by a
+connector accessing a data source. They are supported by all connectors:
+
+* :doc:`/sql/select` including :doc:`/sql/match-recognize`
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* ``SHOW CREATE MATERIALIZED VIEW``
+* :doc:`/sql/show-create-schema`
+* :doc:`/sql/show-create-table`
+* :doc:`/sql/show-create-view`
+* :doc:`/sql/show-grants`
+* :doc:`/sql/show-roles`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/show-stats`
+* :doc:`/sql/describe`
+
+.. _sql-write-operations:
+
+Write operations
+----------------
+
+The following statements provide write access to data and meta data exposed by
+by a connector accessing a data source. Availability varies widely from
+connector to connector:
+
+.. _sql-write-data:
+
+Data management
+^^^^^^^^^^^^^^^
+
+* :doc:`/sql/delete`
+* :doc:`/sql/insert`
+* :doc:`/sql/update`
+
+.. _sql-write-materialized-views:
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+* ``CREATE MATERIALIZED VIEW``
+* ``DROP MATERIALIZED VIEW``
+* ``REFRESH MATERIALIZED VIEW``
+
+.. _sql-write-table-schema:
+
+Table and schema management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/comment`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/create-table`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+
+.. _sql-write-views:
+
+Views
+^^^^^
+
+* :doc:`/sql/create-view`
+* :doc:`/sql/drop-view`
+* :doc:`/sql/alter-view`
+
+
+.. _sql-security-operations:
+
+Security operations
+-------------------
+
+The following statements provide security-related operations to security
+configuration, data and meta data exposed by a connector accessing a data
+source. Most connectors do not support these operations:
+
+Connector roles:
+
+* :doc:`/sql/create-role`
+* :doc:`/sql/drop-role`
+* :doc:`/sql/grant-roles`
+* :doc:`/sql/revoke-roles`
+* :doc:`/sql/set-role`
+* :doc:`/sql/show-role-grants`
+
+Grants management:
+
+* :doc:`/sql/grant`
+* :doc:`/sql/revoke`
+
+.. _sql-security-transactions:
+
+Transactions
+------------
+
+The following statements manage transactions. Most connectors do not support
+transactions. Details are available with the documentation for each
+connector:
```

Comment:
"Details are available with the documentation for each connector:"

If we're including this snippet here, we should include it with all the other section introductions that say that support is limited to some connectors. If we can't do that, because not all the connector docs have a list of supported SQL statements yet, then I don't think we should have this line at all until that's the case.

Reply:
You also have a nearly identical sentence above all of these sections at line 25, so you could just remove this from here as redundant.

Reply:
good point... made it all consistent now